packages/intl-getcanonicallocales/index.ts

```diff
@@ -1,6 +1,6 @@
-import {parseUnicodeLocaleId} from './parser';
-import {emitUnicodeLocaleId} from './emitter';
-import {canonicalizeUnicodeLocaleId} from './canonicalizer';
+import {parseUnicodeLocaleId} from './src/parser';
+import {emitUnicodeLocaleId} from './src/emitter';
+import {canonicalizeUnicodeLocaleId} from './src/canonicalizer';
```

Comment:
Why move source files to root directory. Is it because Bazel is incompatible with `src` folder structure?